cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperCluster.java

```diff
@@ -95,6 +108,10 @@ public static String getPersistentVolumeClaimName(String clusterName, int podId)
         return VOLUME_NAME + "-" + clusterName + "-" + podId;
     }
 
+    public static String nodesSecretName(String cluster) {
```

Comment:
I wasn't convinced about that name as well. I used "brokers" in Kafka cluster because we say "Kafka broker" but then because we say (at leasy me) "Zookeeper nodes" I started to use "nodes" here.

Reply:
I somewhat agree about that, but there are pros and cons to that. If we call the kafka brokers nodes, then we will often have to disambiguate ("Kafka nodes" vs "Zookeeper nodes"), And node is already an overloaded term (K8s nodes). Really these things are containers from the k8s pov, so maybe that's what we should call them: kafka containers and zk containers?

Reply:
Well ... its Kafka broker nodes as well, right ... I think nodes could be used everywhere and it would be a bit more clear that this is the same thing.

Reply:
and what if some day we'll have other containers (I'm not referring to the stunnel ones) where some certs could be needed? Or am I thinking too much about unknown future :-)

Reply:
We haven't a consensus here so I'm going to leave things as they are. It's really a nit, we can change later with just one commit.